srcpkgs/eudev/template

```diff
@@ -3,8 +3,8 @@
 _UDEV_VERSION="228" # compatible udev version provided
```

Comment:
configure.ac defines 
```
AC_SUBST(UDEV_VERSION, 243)
```

so I think this should also be updated here.

Reply:
thanks for catching! pushed.